BrainPortal/public/javascripts/cbrain.js

```diff
@@ -819,31 +686,31 @@
       form.submit();
     });
 
-    
+
     //html_tool_tip_code based on xstooltip provided by
     //http://www.texsoft.it/index.php?%20m=sw.js.htmltooltip&c=software&l=it
     $(document).delegate(".html_tool_tip_trigger", "mouseenter", function(event) {
       var trigger = $(this);
-      var tool_tip_id = trigger.attr("data-tool-tip-id");
+      var tool_tip_id = trigger.data("tool-tip-id");
       var tool_tip = $("#" + tool_tip_id);
-      var offset_x = trigger.attr("data-offset-x") || '30';
-      var offset_y = trigger.attr("data-offset-y") || '0';
-      
+      var offset_x = trigger.data("offset-x") || '30';
+      var offset_y = trigger.data("offset-y") || '0';
+
       var x = trigger.position().left + parseInt(offset_x, 10);
       var y = trigger.position().top  + parseInt(offset_y, 10);
-      
+
       // Fixed position bug.
       tool_tip.remove().appendTo(trigger.parent());
```

Comment:
This hack causes weird problems when the parent has fancy CSS and is due to using improper CSS attributes on the tooltip itself. This entire handler should be made _much_ shorter and to the point using jQuery's `offset` method.